presto-benchto-benchmarks/README.md

```diff
@@ -0,0 +1,72 @@
+# Presto Benchto benchmarks
+
+The Benchto benchmarks utilize [Benchto](https://github.com/prestodb/benchto) benchmarking
+utility to do macro benchmarking of Presto. As opposed to micro benchmarking which exercises
+a class or a small, coherent set of classes, macro benchmarks done with Benchto use Presto
+end-to-end, by accessing it through its API (usually with `presto-jdbc`), executing queries,
+measuring time and gathering various metrics.
+
+## Benchmarking suites
+
+Even though benchmarks exercise Presto end-to-end, a single benchmark cannot use all Presto
+features. Therefore benchmarks are organized in suites, like:
+
+* *tpch* - queries closely following the [TPC-H](http://www.tpc.org/tpch/) benchmark 
+* *tpcds* - queries closely following the [TPC-DS](http://www.tpc.org/tpcds/) benchmark 
+
+## Usage
+
+### Requirements
+
+* Presto already installed on the target environment
+* Basic understanding of Benchto [components and architecture](https://github.com/prestodb/benchto)
+* Benchto service [configured and running](https://github.com/prestodb/benchto/tree/master/benchto-service)
+* An environment [defined in Benchto service](https://github.com/prestodb/benchto/tree/master/benchto-service#creating-environment)
+
+### Configuring benchmarks
+
+Benchto driver needs to know two things: what benchmark is to be run and what environment
+it is to be run on. For the purpose of the following example, we will use `tpch` benchmark
+and Presto server running at `localhost:8080`, with Benchto service running at `localhost:8081`.
+
+Benchto driver uses Sprint Boot to locate environment configuration file, so to pass the
+configuration. To continue with our example, one needs to place an `application-presto-devenv.yml`
```

Comment:
Now I see that it works through spring. Is `application.yaml` (please fix this here! -- you have yml) is read by spring and merged with `application-presto-devenv.yml` by spring? In this case, application.yaml is read, then application-presto-devenv.yaml and merged?